lib/commands/project/project.js

```diff
@@ -95,6 +95,20 @@ function create(name, options, cb) {
     cloneSkeleton(name, framework, targetDir, function(err) {
       if (err) { return cb(err); }
       emit('Project %s created in %s', name, targetDir);
+
+      findProjectFile(targetDir, null, function(err, projPath) {
+        var projectFile = fs.readFileSync(projPath, 'utf8');
+        var projectJson = JSON.parse(projectFile);
+
+        projectJson.devDependencies['z-schema'] = '^3.12.0';
```

Comment:
if user never uses generate-test command the package.json should not have those dependencies. We do need to run `npm i` twice but I was suggesting to encapsulate the code that runs `npm i` into a function and call it here and when generate-test is called.